ktor-http/src/io/ktor/http/ContentTypes.kt

```diff
@@ -216,4 +217,4 @@ fun ContentType.withCharset(charset: Charset) = withParameter("charset", charset
 /**
  * Extracts a [Charset] value from the given `Content-Type`, `Content-Disposition` or similar header value.  
  */
-fun HeaderValueWithParameters.charset() = parameter("charset")?.let { Charset.forName(it) }
\ No newline at end of file
+fun HeaderValueWithParameters.charset() = parameter("charset")?.let { Charset.forName(it) }
```

Comment:
I created this PR with the github UI (didn't even check out the code locally). It seems to automatically add these when you do that. Sorry if this is an annoyance.